index.js

```diff
@@ -474,6 +474,9 @@ class MillerColumnsElement extends HTMLElement {
         } else if (showNarrow) {
           item.classList.add(narrowClass)
         }
+        if (columnsToShow.length === 0) {
+          item.classList.add(activeClass)
```

Comment:
Probably worth writing a unit test for this edge case too 👍

Reply:
Mmm, it's captured already, [checking the first column is active by default](https://github.com/alphagov/miller-columns-element/pull/25/files#diff-c1129c8b045390789fa8ff62f2c6b4a9R250).

Reply:
Ideally that line of code n the test would be added in the same commit as this one 👍 as otherwise I imagine the tests would have been failing at that point.

Reply:
Agree, when I split the commit I should've split the test too. I now moved the assertion in the appropriate commit.